app/lib/frontend/keys.dart

```diff
@@ -24,3 +24,19 @@ Future<String> customSearchKeyFromDB() async {
 
   return apiKey.value;
 }
+
+String _cachedNotificationSecret;
+
+/// Gets the shared notification secret from the datastore (or local cache).
+///
+/// Ignoring cache invalidations for now, because at worst the notification is
+/// not acknowledged, and the related processing will happen 10 minutes later.
+Future<String> getNotificationSecret() async {
+  if (_cachedNotificationSecret == null) {
+    final db = dbService;
+    final key = db.emptyKey.append(PrivateKey, id: 'notification-secret');
```

Comment:
Please put this constant somewhere else and validate that the `id` used in the command line tool is this one. This avoids setting accidentally the wrong `id` (aka key).